app/models/volunteer.rb

```diff
@@ -155,12 +155,19 @@ def in_region?(region_id)
 
   def current_absences
     today = Date.today
-
     absences.where('start_date < ? AND stop_date > ?', today, today)
   end
 
+  def region_names
+    assignments.flat_map { |a| a.region.try(:name) }.uniq.join(",")
+  end
+
   ### CLASS METHODS
 
+  def self.with_regions_for_select(vols)
+    vols.collect { |v| ["#{v.name} ['#{v.region_names}']", v.id] }
+  end
```

Comment:
would be awesome to have a unit test for this method

Reply:
Updated